Example/ScalingCarousel/CodeViewController.swift

```diff
@@ -76,7 +90,11 @@ extension CodeViewController: UICollectionViewDataSource {
         if let scalingCell = cell as? ScalingCarouselCell {
             scalingCell.mainView.backgroundColor = .blue
         }
-        
+        DispatchQueue.main.async {
```

Comment:
Why is this code needed?

Reply:
In the func deviceRotated() I have set another DispatchQueue.main.async, if not it doesn't work fine, I have debugged the deviceRotated without setting DispatchQueue.main.async and it says that the code is running in the main thread, but it doesn't work properly, with the dispatch of the deviceRotated is enough and the dispatch of cell.setNeedsLayout() is not necessary, but maybe as just double check that any layout of the cell gets executed in main thread doesn't hurt. (Also in my project to hide the carousel I need to force it to get executed in main queue even if when I debug it already says that is in main queue) https://stackoverflow.com/questions/50547386/swift-animate-duration-not-working-in-cgaffinetransform/50547387#50547387. Maybe this happens for a different reason but I don't know why :S hahahaha